### PR TITLE
Nosql and c++, Both Formatting

### PR DESCRIPTION
**Pull request by tik9**
Again had to delete two lines regarding conflict in c++

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,76 +7,85 @@
...
+- [Javascript](javascript.md)
+- [Mysql](mysql.md)
+- [Powershell](powershell.md)
+- [Python Core](python_core.md)
+- [Python Data Analysis](python_data_analysis.md)
+- [Python applied Data Mining](python_applied_data_mining.md)
+- [Python Web Scraping](python_web_scraping.md)
+- [Python Exploratory Data Analysis](python_exploratory_data_analysis.md)
+- [Python Clean Data](python_clean_data.md)
+- [Python Data Visualization](python_data_visualization.md)
```
Comment (Ebazhanov) [inline README.md:28]: please revert back to the table format, it looks more organize and beautiful in my opinion! Thank you in advance!

File: nosql/nosql-quiz.md [hunk 2]
```diff
@@ -1,280 +1,295 @@
...
-    - values
-    - scroes <<<<----Correct
-    - ids
 
-Q23. You need to create a document database that supports database triggers. Which NoSQL database should you use?
+#### Q3. To index a field that holds an array value in MongoDB, which index type do you create? 
+- [] partial 
+- [] sparse 
+- [] compound 
+- [x] multikey <<<<<--CORRECT
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:22]:
```suggestion
- [x] multikey
```

File: nosql/nosql-quiz.md [hunk 3]
```diff
@@ -1,280 +1,295 @@
...
-    - GraphDB
-    - DynamoDB
-    - Cosmos DB <<<<----Correct
 
-Q27. You need to control your application's batch updates destination for your Bigtable multinode cluster. Which action do you take?
+#### Q7. You need to design security for DynamoDB to allow users read-only access to certain items and attributes in a table. What do you do? 
+- [] Use lAM policy conditions 
+- [] Use lAM roles 
+- [] Use VPC endpoint 
+- [x] Use lAM policies <<<<<--CORRECT
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:50]:
```suggestion
- [x] Use lAM policies
```

File: nosql/nosql-quiz.md [hunk 4]
```diff
@@ -1,280 +1,295 @@
...
-    - Create a custom app profile to route the batch update from that client.
-    - Update the default app profile to route the natch update from that client. <<<<--- May be
-    - Use the default app profile to route batch updates.
 
-Q29. Polyglot storage mean using multiple types of what in the same application?
+#### Q8. What does redis use to sort the elements of a sorted set? 
+- [] Scores. 
+- [] Ids. 
+- [] Values. 
+- [x] Keys. <<<<<--CORRECT
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:57]:
```suggestion
- [x] Keys.
```

File: nosql/nosql-quiz.md [hunk 5]
```diff
@@ -1,280 +1,295 @@
...
-    - The queried key value exists, but has no associated expire value.
-    - The queried key value does not exist. <<<<----Correct
-    - There are two expired keys with this value.
 
-Q31. You are designing a MongoDB schema to support queries that will include lookups. What should you do?
+#### Q10. Polyglot storage means using multiple types of what in the same application? 
+- [] Security systems. 
+- [] Database systems. 
+- [] Storage systems. 
+- [x] Query systems. <<<<<--CORRECT
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:71]:
```suggestion
- [x] Query systems.
```

File: nosql/nosql-quiz.md [hunk 6]
```diff
@@ -1,280 +1,295 @@
...
-    - Create an index on the key value used as the foreign key.
-    - Create a multicolumn index on the key value used as the foreign key and the most unique column in the document. <<<<----Correct
-    - Create a multicolumn index on the key value used as the primary and also the forign key.
 
-Q32. What is the aggregation operator for a join concept in MongoDB?
+#### Q11. You need to implement the simplest possible scalable, in-memory cache for your AWS application. Which service do you select? 
+- [] Elasticache using Memcached. 
+- [] DynamoDB. 
+- [] DynamoDB Accelerator (DAX). 
+- [x] Elasticache using Redis. <<<<<--CORRECT
```
Comment (Ebazhanov) [inline nosql/nosql-quiz.md:78]:
```suggestion
- [x] Elasticache using Redis.
```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
left some comments + general problem:
- no empty space inside the braces `[]` => `[ ]`
official documents https://guides.github.com/features/mastering-markdown/
```
- [x] this is a complete item
- [ ] this is an incomplete item
```

and as usual, your PR's are too big to have a quick review, so I would suggest you make it smaller.  FYI interesting to read https://essenceofcode.com/2019/10/29/the-art-of-small-pull-requests/

File: README.md (hunk 1 above)
Comment (tik9) [reply README.md:28]: Yes, I did that but it got lost in another PR. Do it again.

File: README.md [hunk 7]
```diff
@@ -7,76 +7,85 @@
...
+| [Scala](scala/scala-quiz.md)                                                 | answers required               |
+| [Search Engine Optimization (SEO)](seo/search-engine-optimization-quiz.md)   | partially answered             |
+| [SharePoint](sharepoint/sharepoint-quiz.md)                                  | with answers                   |
+| [Swift](swift/swift-quiz.md)                                                 | with answers                   |
+| [Transact-SQL (T-SQL)](t-sql/t-sql-quiz.md)                                  | with answers                   |
+| [VBA](vba/vba-quiz.md)                                                        | answers required             |
+
+| [Windows Server](windows-server/windows-server-quiz.md)                      | partially answered             |
+| [WordPress](wordpress/wordpress-quiz.md)                                     | partially answered             |
+| [XML](xml/xml-quiz.md)                                                       | answers required               |
```
[the remark this answers is not in the document]
Comment (tik9) [reply README.md:88]: I agree of making the PR smaller. There is some issue with my git, see:
image
When I checkout master, only nosql/nosql-quiz.md should be committed, but not READme. Why? I expect README to also come up here, not only nosql. Is this the problem of my huge PR? I always try to have small PRs, just one skill test.

[the remark this answers is not in the document]
Comment (Ebazhanov) [reply README.md]: let's go to our private discussion, I am quite sure we can solve it! :punch:

**Comment by tik9** [comment]
Now you can merge.

**Pull request merged**